openapi/vimeo/Module.md

```diff
@@ -39,14 +39,14 @@ You can do this step in two ways. You can use any one of this.
 
     1. Set up token in `Config.toml` as shown below.
     ```
-    [auth]
+    [authConfig]
     token = "<Enter your token here>"
     ```
 
     2. Configure the client in ballerina file as shown below.
     ```ballerina
-    configurable http:BearerTokenConfig & readonly auth = ?;
-    vimeo:ClientConfig clientConfig = {auth : auth};
+    configurable http:BearerTokenConfig & readonly authConfig = ?;
```

Comment:
The field name should to be `auth`

Please check how record was generated.
```
    http:OAuth2ClientCredentialsGrantConfig|http:BearerTokenConfig|http:OAuth2RefreshTokenGrantConfig auth;

```

Reply:
Sure. Let's change.